distributed/src/test/scala/asobu/distributed/util/package.scala

```diff
@@ -0,0 +1,14 @@
+package asobu.distributed
+
+import asobu.distributed.CustomRequestExtractorDefinition.Interpreter
+import asobu.dsl._
+
+package object util {
+
+  object implicits {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Object name does not match the regular expression '^[A-Z][A-Za-z0-9]*$'](https://www.codacy.com/app/iHeartRadio/asobu/file/2681756979/issues/source?bid=3037259&fileBranchId=3335316#l8)